pdc_client/__init__.py

```diff
@@ -269,6 +270,31 @@ def __getattr__(self, name):
     def __getitem__(self, *args, **kwargs):
         return _BeanBagWrapper(self.client.__getitem__(*args, **kwargs), self.page_size)
 
+    def results(self, *args, **kwargs):
+        """
+           Return an iterator with all pages of data when multi pages.
+           Return a list when there is a single page.
```

Comment:
This is misleading since the return type is always an iterator (not a list).

I think this must be documented elsewhere (e.g. usage examples in docstring for `PDCClient`), user shouldn't know that there is `_BeanBagWrapper` private class ([API docs for private stuff is not generated](https://product-definition-center.github.io/pdc-client/api.html)).

Reply:
Yep, will change that.